lib/utils/__tests__/addSemicolonForEditInfo.test.mjs

```diff
@@ -0,0 +1,189 @@
+import postcss from 'postcss';
+
+import addSemicolonForEditInfo from '../addSemicolonForEditInfo.mjs';
+
+describe('addSemicolonForEditInfo', () => {
+	describe('declaration', () => {
+		test('in the middle of a block having no trailing semicolon', () => {
+			const node = postcss.parse('a { color: red; top: 0px }').first.first;
+
+			const fixData = {
+				range: [0, 0],
+				text: node.toString(),
+			};
+
+			expect(fixData.text).toBe('color: red');
+
+			const text = addSemicolonForEditInfo(node, fixData).text;
+
+			expect(text).toBe('color: red;');
```

Comment:
[suggestion] I'd like to test the whole return value of `addSemicolonForEditInfo` to ensure that `range` is unchanged. How about this? (as well as other test cases)

```suggestion
			expect(addSemicolonForEditInfo(node, fixData)).toEqual({
				range: [0, 0],
				text: 'color: red;',
			});
```

Reply:
Done!